charts/linkerd2-cni/values.yaml

```diff
@@ -20,6 +20,12 @@ logLevel:         info
 portsToRedirect:  ""
 # -- User id under which the proxy shall be ran
 proxyUID:         2102
+# -- Optional customisation of the group id under which the proxy shall be ran
```

Comment:
I think we need to document this a bit more to prevent people from misusing the field.

1. We should note that proxy's GID should be unique. No other container should run under the same group as the proxy (including the control plane's containers). Once the group ID is shared, packets can be skipped for processes that are not the proxy and it will result in hard to diagnose issues.
2. We should consider using the `fail` function to fail-fast if the proxy's gid == controller gid.